apps/arweave/src/ar.erl

```diff
@@ -664,6 +665,18 @@ parse_cli_args(["rocksdb_flush_interval", Seconds | Rest], C) ->
 parse_cli_args(["rocksdb_wal_sync_interval", Seconds | Rest], C) ->
 	parse_cli_args(Rest, C#config{ rocksdb_wal_sync_interval_s = list_to_integer(Seconds) });
 
+%% shutdown procedure
+parse_cli_args(["shutdown_tcp_connection_timeout", Delay|Rest], C) ->
```

Comment:
For now I think probably best to keep to the format of the other parse lines, eg.

```
parse_cli_args(["rocksdb_flush_interval", Seconds | Rest], C) ->
	parse_cli_args(Rest, C#config{ rocksdb_flush_interval_s = list_to_integer(Seconds) });
```

This file is already monstrous enough, but keeping the styling/format the same helps a little.

Plus in general we do a hard/fast fail if any of the flags are set wrong rather than continue processing (e.g. if list_to_integer fails, we should allow the node to exit as we do for the other flags0